app/controllers/health_check_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class HealthCheckController < ActionController::Base
+  def health
+    response = { version: ENV.fetch('VERSION'), status: 'UP' }
```

Comment:
Did you try it?

I don't understand how it would work, since `VERSION` is never set at runtime?